tsp_cli_client

```diff
@@ -428,7 +428,7 @@ if __name__ == "__main__":
             if options.type_id is not None:
                 response = tsp_client.fetch_configuration(options.type_id, options.list_configuration)
                 if response.status_code == 200:
-                    print('  {0}'.format(response.model.to_string()))
+                    print(f'  {response.model}
```

Comment:
this leads to an error `SyntaxError: unterminated string literal (detected at line 431)` 
You need to terminate the string